### ISSUE TITLE
Support S3

### ISSUE THREAD

**Issue by joshmanders** [aniftyco/kubit#39]

**Comment by dmdboi** [comment]
Im assuming this is out of the box without needing a 3rd-party extension. Similarly, this should support Cloudflare R2

**Comment by joshmanders** [comment]
That is correct, instead of installing a first party package for doing s3, we just expect you to install `aws-sdk` as a peer dependency and it just works.
